tests/unit/test_network_connections.py

```diff
@@ -445,6 +448,8 @@ def test_validate_list(self):
         )
         self.assertEqual([1, 5], v.validate(["1", 5]))
         self.assertValidationError(v, [1, "s"])
+        # Assert that `None` is normalized into `[]`` in ArgvalidatorList()
+        self.assertEqual(v.validate(None), [])
```

Comment:
ideally, these changes would be in the commits that introduce this behavior respectively.

Reply:
I already put these changes in the commits that introduce this behavior respectively. Thanks.

Reply:
I already put these changes in the commits that introduce this behavior respectively. Thanks.